nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -232,9 +232,12 @@ in
         '';
       };
 
+      # These values are merged with the ones defined externally, see:
+      # https://github.com/NixOS/nixpkgs/pull/10155
+      # https://github.com/NixOS/nixpkgs/pull/41745
```

Comment:
No they aren't! If those are option-defaults, those will be lost when defining `authorizedKeysFiles`.